frontend/src/components/atoms/TextField.tsx

```diff
@@ -1,8 +1,51 @@
 import React from "react";
+import { TextField } from "@mui/material";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
 
-/** A TextField page */
-const TextField = () => {
-  return <>Hello there</>;
+interface Props {
+  label: string;
+  required: boolean;
+  status: string; //error | ""
+  //if required, have * next to entry ?
+  //if error, display incorrect entry text
+  //onChange:
+  incorrectEntryText: string;
```

Comment:
We are thinking of using a form library for form validation so this prop might be irrelevant. You can keep it for now but there is a high chance it will be useless as soon as we implement form validation